### DISCUSSION TITLE
Applying interactivity to non-player characters

### DISCUSSION THREAD

**Discussion by AlexP-Coding** [MixedRealityToolkit/MixedRealityToolkit-Unity#599] [Q&A]
## Describe the issue
Hello! 
I am looking give interaction capabilities to an NPC within a game, just as I would a real, human player. However, I'm not sure how to apply to this humanoid NPC the same properties that allow the human player to interact with objects - be it by gaze or by grabbing. I've been looking into the documentation, but it focuses much more on the object's end of things (which - fair).

How would I send the information that the NPC is grabbing a cup, for example? I've read the architecture but I still don't understand how we might be able to let an NPC interact with the environment as if they were human.

## Feature area
Information needed on how to emulate a player profile (with the ability to interact with objects) on an NPC.

## Additional context
Device (if relevant): HoloLens 2.
MRTK version: MRTK3
Software: Unity 2021.3

**Comment by AMollis** [comment]
Moving this to the discussion forums.  This doesn't sound like a bug.

**Reply by AlexP-Coding** [reply]
It's not, which is why it was marked as a documentation issue. I was unsure where it fit best, so thank you for clarifying.
